docs/docs/casper/storage.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Storage
+
+slug: /casper/storage
+---
+
+This section provides an in-depth exploration of key data structures integral to the Invariant protocol's storage mechanism. These structs are specifically crafted to facilitate the sharing of the state of the exchange within the CLAMM model. These data structures play a pivotal role in maintaining and organizing information related to the exchange, ensuring efficient and organized handling of data.
+
+## Contract State
+
+```rust
+#[derive(OdraType)]
+pub struct State {
+    pub admin: Address,
+    pub protocol_fee: Percentage,
+}
+```
+
+| Name         | Type       | Description                                                                                                                                                                                 |
+| ------------ | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| admin        | Address    | Account address of protocol admin. Admin is able to change fee, claim protocol fee or set the fee receiver, but cannot interfere with user positions or deposits and cannot close the pool. |
+| protocol_fee | Percentage | Percentage of the fee collected upon every swap in the pool that goes to the protocol, rest goes to LP.                                                                                     |
+
+## FeeTier
+
+```rust
+#[derive(OdraType)]
+pub struct FeeTier {
+    pub fee: Percentage,
+    pub tick_spacing: u16,
```

Comment:
Make sure all structs align with CSPR. Here, `tick_spacing` should be of type `u32`